README.md

```diff
@@ -42,56 +42,66 @@ yarn add @ably/asset-tracking
 
 ## Usage
 
-Here is an example of how the SDK can be used:
+### Subscribing to location updates
 
 ```ts
-import { Subscriber, Accuracy } from '@ably/asset-tracking';
+import { Subscriber } from '@ably/asset-tracking';
 
+// Client options passed to the underling ably-js instance
+// You must provide a clientId and some way for the client to authenticate with Ably
```

Comment:
This is slightly misleading, considering token auth can include the client identifier in the token. Also, I think it could be a little more friendly. Perhaps...

```suggestion
// You must provide some way for the client to authenticate with Ably.
// In this example we're using basic authentication which means we must also provide a clientId.
// See: https://ably.com/docs/core-features/authentication
```

Reply:
very good point, fixed 😄